app/blog/2020-09-22-lit-element-and-lit-html-next-preview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "LitElement 3.0 & lit-html 2.0: Early Preview Release"
+---
+
+Today we're publishing the very firse preview releases of the next major versions of our flagship libraries, LitElement and lit-html.
+
+These releases are not yet feature complete or fully API stable, but they include most of the breaking changes that we intend to make, and most of the functionality we want to carry over from the previous versions. The releases also do not yet support legacy browsers like IE11, or browsers requiring the web components polyfills.
+
+## Motivation
+
+We've been very happy with the current version of our libraries - they're fast, small, and stable (yay!) - and in some ways we _don't_ have very many pressing needs to make breaking changes. We don't take making breaking changes lightly. But there are some compelling reasons for changes that we think will improve the user-experience of components and applications built with LitElement.
+
+1. Performance. We have found that some of our browser-bug workaround code and customization abstractions prevent optimizations that we would like to do.
+2. Size. That same code, and our extensive public API, costs bytes. We always want to find ways to make the liraries smaller.
+3. Features & API cleanup. Some features are difficult to add in a cost-effective way with the current architecture, or can't really be improved without small tecnically breaking change.
```

Comment:
```suggestion
3. Features & API cleanup. Some features are difficult to add in a cost-effective way with the current architecture, or can't really be improved without breaking changes.
```
Not sure what "technically" meant in this sentence.

Reply:
just that they're breaking changes that probably don't break anyone